e2e/pages/adf/demo-shell/process-services/tasksCloudDemoPage.ts

```diff
@@ -94,6 +94,10 @@ export class TasksCloudDemoPage {
         return BrowserActions.getText(this.activeFilter);
     }
 
+    async isFilterActive(filterName) {
```

Comment:
BrowserVisibility.waitUntilElementIsVisible will never return false. In case the element will not be visible, it will throw error so it makes no sense to try to return it and use it in expect.